lib/expansion_rules.rb

```diff
@@ -76,13 +76,28 @@ def details_fields(*fields)
   TAXON_FIELDS = (DEFAULT_FIELDS + %i[description details phase]).freeze
   NEED_FIELDS = (DEFAULT_FIELDS + details_fields(:role, :goal, :benefit, :met_when, :justifications)).freeze
   FINDER_FIELDS = (DEFAULT_FIELDS + details_fields(:facets)).freeze
-  PERSON_FIELDS = (DEFAULT_FIELDS + details_fields(:body)).freeze
+  PERSON_FIELDS_WITH_BODY = (DEFAULT_FIELDS + details_fields(:body)).freeze
+  PERSON_FIELDS_WITH_IMAGE = (DEFAULT_FIELDS + details_fields(:image)).freeze
   ROLE_FIELDS = (DEFAULT_FIELDS + details_fields(:body)).freeze
   ROLE_APPOINTMENT_FIELDS = (DEFAULT_FIELDS + details_fields(:started_on, :ended_on, :current, :person_appointment_order)).freeze
   STEP_BY_STEP_FIELDS = (DEFAULT_FIELDS + [%i[details step_by_step_nav title], %i[details step_by_step_nav steps]]).freeze
   STEP_BY_STEP_AUTH_BYPASS_FIELDS = (STEP_BY_STEP_FIELDS + %i[auth_bypass_ids]).freeze
   TRAVEL_ADVICE_FIELDS = (DEFAULT_FIELDS + details_fields(:country, :change_description)).freeze
   WORLD_LOCATION_FIELDS = %i[content_id title schema_name locale analytics_identifier].freeze
+
+  CUSTOM_EXPANSION_FIELDS_FOR_PEOPLE = (
+    %i(
+      ordered_ministers
+      ordered_board_members
+      ordered_military_personnel
+      ordered_traffic_commissioners
+      ordered_chief_professional_officers
+      ordered_special_representatives
```

Comment:
We'll also need the image for `role_appointments` for /government/ministers, though I can add that in my PR if this one gets merged first

Reply:
I've added it to this PR in 3b4cd2f41fda219f346329fbb367b8c2c983c747.